scripts/build-android.js

```diff
@@ -51,24 +51,24 @@ if (options.arch) {
 
 const buildType = options.buildType;
 
-const ndkPath = process.env["ANDROID_NDK"];
+const ndkPath = process.env["ANDROID_NDK"] || process.env["ANDROID_NDK_HOME"];
```

Comment:
Why is this needed?

Reply:
The NDK path in github actions is stored in ANDROID_NDK_HOME